testsuite/test_interface.py

```diff
@@ -30,16 +38,35 @@ def test_submission(client):
         )
 
     assert len(Submission.objects.all()) == 1
-    assert storage.exists('1.zip')
+    assert storage.exists('1.zip') is True
```

Comment:
Q: Shouldn't be ok only with ```assert storage.exists(`1.zip`)```?

Reply:
Good point. I guess I am stuck with a bit of verbosity from C